src/components/paper-tabs/paper-tabs.js

```diff
@@ -0,0 +1,663 @@
+/**
```

Comment:
Don't add a copy of `paper-tabs`, but create a `ha-paper-tabs` that extends `paper-tabs` and adjust the things that need to be adjusted.